packages/expo-blur/android/src/main/java/expo/modules/blur/enums/BlurMethod.kt

```diff
@@ -0,0 +1,8 @@
+package expo.modules.blur.enums
+
+import expo.modules.kotlin.types.Enumerable
+
+enum class BlurMethod(val method: String) : Enumerable {
+  NONE("NONE"),
+  DIMEZIS_BLUR_VIEW("DIMEZIS_BLUR_VIEW")
+}
```

Comment:
```suggestion
enum class BlurMethod(val value: String) : Enumerable {
  NONE("none"),
  DIMEZIS_BLUR_VIEW("dimezisBlurView")
}
```
The constructor argument needs to be named `value`